.github/workflows/ci_cd.yml

```diff
@@ -158,10 +158,22 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           doc-artifact-name: documentation-html
 
+  update-changelog:
+    name: "Update CHANGELOG for new tag"
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: ansys/actions/doc-deploy-changelog@v6
+        with:
+          token: ${{ '{{ secrets.PYANSYS_CI_BOT_TOKEN }}' }}
```

Comment:
This seems strangely indirect. Is this required?

Reply:
https://actions.docs.ansys.com/version/stable/migrations/docs-deploy-changelog-setup.html#docs-deploy-changelog-action-setup copied verbatim from the migration guide

Reply:
Checked with Kerry and it's an oversight